src/styles/global/header.scss

```diff
@@ -1,3 +1,155 @@
-header {
-  color: $water;
+.nav-menu {
+  width: 100%;
+  height: 64px;
+}
+
+.nav-menu ul {
+  list-style: none;
+  overflow: hidden;
+}
+
+.nav-menu li a {
+  display: block;
```

Comment:
`.d-block` funkar kanske inte?

Reply:
removed display block since it does nothing